applications/Unity.GrantManager/src/Unity.GrantManager.Domain/Unity.GrantManager.Domain.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Unity.GrantManager.Application.Contracts\Unity.GrantManager.Application.Contracts.csproj" />

```

Comment:
The domain project shouldn't reference the application contracts project - looking at how to refactor this

Reply:
@jimmyPasta - please see branch:
https://github.com/bcgov/Unity/tree/feature/AB%238819-Applicant-Mapping-Refactor
I tried to rework the code to keep the domain layer clean and not reference the contracts project, will need some testing to make sure the logic is still sound

Reply:
Ok Thanks Should I merge 238819 into this one?

Reply:
yeah, if you are happy with the changes and i didn't break anything